triton-client/src/components/ProjectDiskUsage/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { FunctionComponent, useEffect } from "react"
+import { selectConstants } from "@store/constants"
+import { useAppDispatch, useAppSelector } from "@store/hooks"
+import { fetchConstants } from "@store/thunks"
+import { Col, Flex, Layout, Progress, Row } from "antd"
+import { SUPPORTED_DOWNLOAD_TYPES } from "../../constants"
+import { dataSize } from "../../functions"
+import { ProjectDiskUsageProps } from "./interfaces"
+import { Content } from "antd/es/layout/layout"
+import { Typography } from "antd"
+import "./index.scss"
+const { Text, Title } = Typography
+
+const ProjectDiskUsage: FunctionComponent<ProjectDiskUsageProps> = ({
+    projectExternalId,
+}) => {
+    const dispatch = useAppDispatch()
+    const constants = useAppSelector(selectConstants)
+
+    useEffect(() => {
+        dispatch(fetchConstants())
+    }, [dispatch])
+
+    const diskUsage = useAppSelector(
+        (state: any) =>
+            state.projectsState.projectsById[projectExternalId]?.diskUsage,
+    )
+    return (
+        <>
+            {SUPPORTED_DOWNLOAD_TYPES.map((type) => {
+                const usage = diskUsage ? diskUsage[type] : 0
+                const capacity = constants.diskCapacity[type]
+                const [scaledUsage, usageUnit] = diskUsage
+                    ? dataSize(diskUsage[type])
+                    : []
+                const [scaledCapacity, capacityUnit] = dataSize(
+                    constants.diskCapacity[type],
+                )
+                return (
+                    <Layout>
+                        <Content className="disk-type-usage-container">
+                            <Title level={5}>{type}</Title>
```

Comment:
The download type seems a bit too shifted to the right
![image](https://github.com/user-attachments/assets/a9c03a6c-a54d-4486-b098-a1c15c32dd1d)

Reply:
i dont see what you mean, the value is not the same?

Reply:
no the positioning, it seems a bit too far from the sidebar

Reply:
i cant recreate this bug, lets sit down tomorrow and look into it